src/main/java/org/spongepowered/api/text/title/Title.java

```diff
@@ -204,4 +207,390 @@ public String toString() {
                 .toString();
     }
 
+    /**
+     * Represents a builder class to create immutable {@link Title}
+     * configurations.
+     *
+     * @see Title
+     */
+    public static class Builder {
+
+        @Nullable protected Text title;
+        @Nullable protected Text subtitle;
+        @Nullable protected Integer fadeIn;
+        @Nullable protected Integer stay;
+        @Nullable protected Integer fadeOut;
+        protected boolean clear;
+        protected boolean reset;
+
+        /**
+         * Constructs a new empty {@link Builder}.
+         */
+        public Builder() {
+        }
+
+        /**
+         * Constructs a new {@link Builder} with the properties of the given
+         * {@link Title} as initial values.
+         *
+         * @param title The title to copy the values from
+         */
+        public Builder(Title title) {
```

Comment:
Why are the constructors public? Shouldn't devs use the toBuilder() and static builder() methods?